### ISSUE TITLE
Static Content Doesn't Render on Server

### ISSUE THREAD

**Issue by ejmg** [mmistakes/minimal-mistakes#637]
- [x] This is a question about using the theme.
- [x] I believe this to be a bug with the theme --- not Jekyll, GitHub Pages or one of the bundled plugins.
- [ ] This is a feature request.
- [ ] I have updated all gems with `bundle update`.
- [ ] I have tested locally with `bundle exec jekyll build`.

---

## Environment informations

  Theme Version: 4.0, Jekyll 3.3, Ruby 2.3, Raspbian Jessie on Raspberry Pi B+, Nginx 1.6.2

## Expected behavior

Static Content is presented as expected/tested on local machine, links are functional -> website renders poorly, most js/css functionality is absent, links result in faulty URLs

## The Problem/Question

![screenshot from 2016-11-06 04 00 59](https://cloud.githubusercontent.com/assets/18218174/20036714/a778971e-a3d5-11e6-8443-45508bc89dd0.png)

![screenshot from 2016-11-06 04 02 38](https://cloud.githubusercontent.com/assets/18218174/20036726/e2bced70-a3d5-11e6-8ea0-2c8408b2adcc.png)

I hosted my blog by putting it up on a personal server running nginx (by cloning a pre-generated version), but none of the content/theme renders correctly and with the vast majority of links broken (all links to posts, the about page). It's not that the pages aren't there, it's just that upon clicking a link, for instance "about", the url changes from `ageof.info` to `ageof.info/ageof.info/about` instead of just `ageof.info/about.html`.

I had previously had the boilerplate Jekyll blog running on there with no problem using the same method of pre-generating the static content before cloning it from github. What this means is that there is definitely some problem being created by the added complexity of this theme, but I have absolutely no clue what that is.

I understand there are a lot of variables at play here, including my server, but what I am confused by is that the static content is not generating correctly. I thought once generated through jekyll, the website could be put up for hosting? I have all the other files (_layout/, assets/, etc) also on the server, as well as the necessary gem dependencies using Bundle. Generating the website locally on the server does not fix the problem, either, for both rendering and url redirecting.

I would love any and all help in terms of figuring out the rendering portion of this problem (or any of it, for that matter) so I really appreciate any help I can get. I really like this theme and would love to still host it if possible.

**Comment by mmistakes** [comment]
Check your _config.yml, pretty sure you have `url` improperly. You're missing the protocol. Try:

```yaml
url: http://ageof.info
```

**Comment by ejmg** [comment]
I am **screaming** cries of joy right now. I am so new to webprogramming/server side stuff that I thought I had wasted well over 6 hours on a theme that for some reason or another would not be easily served off of my nginx server. Thank you for giving the time to help me.

The prescribed fix worked like a charm once I rebuilt the site with the edit using `bundle exe jekyll build` and re-uploaded the files for my server to pull and serve, in case anyone else comes across this.

Thanks for an incredibly beautiful theme for my blog, Michael, I'm sure everyone more than appreciates it.
